js/index.js

```diff
@@ -1,42 +1,175 @@
-    "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+  'main-content': {
+    'features-h4': 'Features',
+    'features-content': 'Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'about-h4': 'About',
+    'about-content': 'About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'middle-img-src': 'img/mid-page-accent.jpg',
+    'services-h4': 'Services',
+    'services-content': 'Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'product-h4': 'Product',
+    'product-content': 'Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'vision-h4': 'Vision',
+    'vision-content': 'Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
+  contact: {
+    'contact-h4': 'Contact',
+    address: '123 Way 456 Street Somewhere, USA',
+    phone: '1 (888) 888-8888',
+    email: 'sales@greatidea.io',
   },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  footer: {
+    copyright: 'Copyright Great Idea! 2018',
   },
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+const logo = document.getElementById('logo-img');
+logo.setAttribute('src', siteContent.nav['img-src']);
+
+const headerImage = document.getElementById('cta-img');
+headerImage.setAttribute('src', siteContent.cta['img-src']);
+
+const middleImage = document.getElementById('middle-img');
+middleImage.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+const btn = document.querySelector('button');
+const btn2 = siteContent.cta.button;
+btn.textContent = btn2;
+
+const heading = document.querySelector('h1');
+const heading2 = siteContent.cta.h1;
+heading.textContent = heading2;
+
+const navs = document.querySelectorAll('a');
+const nav1 = siteContent.nav['nav-item-1'];
+
+navs[0].textContent = nav1;
```

Comment:
This is a great place where we could apply the D.R.Y. principle. We could maybe use `forEach` here with some interpolation:
```javascript
navs.forEach((anchor, i) => anchor.textContent = siteContent['nav'][`nav-item-${i + 1}`]);
```